### PR TITLE
start using swagger for manager

### PR DESCRIPTION
**Pull request by yungbender**

### CODE REVIEW

File: manager/docs/docs.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Package docs GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
+// This file was generated by swaggo/swag
+package docs
+
+import "github.com/swaggo/swag"
+
+const docTemplate = `{
+    "schemes": {{ marshal .Schemes }},
+    "swagger": "2.0",
```
Comment (jdobes) [inline manager/docs/docs.go:9]: `github.com/swaggo/swag` supports only OpenAPI 2.0, while we're using OpenAPI 3.0 frameworks in other projects for years already. Not sure if it's a problem.

File: scripts/generate_swagger.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```
Comment (jdobes) [inline scripts/generate_swagger.sh:1]: Why is this script needed when the doc is generated dynamically during docker build? Also why is swagger.json and swagger.yaml in git? When this isn't a spec-first framework they shouldn't be there. It creates extra maintenance steps with every API change to keep them in sync with actual implementation.

**Review by jdobes — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -9,3 +9,8 @@ Vulnerabilities detection for OpenShift images.
     docker-compose down          # Stop and delete containers
     docker-compose down -v       # Stop and delete containers + delete persistent volumes
 
+Manager swagger documentation is running at
+```
+http://localhost:8000/ui/doc.json
```
[the remark this answers is not in the document]
Comment (yungbender) [reply README.md:14]: Changed it, and added a static file endpoint for this specific path for the swagger.json.

File: manager/docs/docs.go (hunk 1 above)
Comment (yungbender) [reply manager/docs/docs.go]: Yes I have noticed that and tried to get inspired from the patchman app, now in the script generate script the 2.0 openapi spec gets transformed to the 3.0 through swagger online converter. The thing is that the generation script is more complicated. I could not find any golang module for generation of 3.0 API. If you have better idea how to do this or wanna write the spec file by ourselves, I am open to new ideas.

**Review by jdobes — APPROVED** [review]

**Pull request merged**